src/contracts/misc/GhoAaveSteward.sol

```diff
@@ -0,0 +1,155 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.10;
+
+import {Ownable} from '@openzeppelin/contracts/access/Ownable.sol';
+import {IPoolAddressesProvider} from '@aave/core-v3/contracts/interfaces/IPoolAddressesProvider.sol';
+import {IPoolConfigurator} from '@aave/core-v3/contracts/interfaces/IPoolConfigurator.sol';
+import {IPool} from '@aave/core-v3/contracts/interfaces/IPool.sol';
+import {DataTypes} from '@aave/core-v3/contracts/protocol/libraries/types/DataTypes.sol';
+import {ReserveConfiguration} from '@aave/core-v3/contracts/protocol/libraries/configuration/ReserveConfiguration.sol';
+import {GhoInterestRateStrategy} from '../facilitators/aave/interestStrategy/GhoInterestRateStrategy.sol';
+import {IFixedRateStrategyFactory} from '../facilitators/aave/interestStrategy/interfaces/IFixedRateStrategyFactory.sol';
+import {IGhoAaveSteward} from './interfaces/IGhoAaveSteward.sol';
+import {RiskCouncilControlled} from './RiskCouncilControlled.sol';
+
+/**
+ * @title GhoAaveSteward
+ * @author Aave Labs
+ * @notice Helper contract for managing parameters of the GHO reserve
+ * @dev Only the Risk Council is able to action contract's functions, based on specific conditions that have been agreed upon with the community.
+ */
+contract GhoAaveSteward is Ownable, IGhoAaveSteward, RiskCouncilControlled {
+  using ReserveConfiguration for DataTypes.ReserveConfigurationMap;
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant GHO_BORROW_RATE_CHANGE_MAX = 0.0500e27; // 5.00%
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant GHO_BORROW_RATE_MAX = 0.2500e27; // 25.00%
+
+  /// @inheritdoc IGhoAaveSteward
+  uint256 public constant MINIMUM_DELAY = 2 days;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable POOL_ADDRESSES_PROVIDER;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable GHO_TOKEN;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable FIXED_RATE_STRATEGY_FACTORY;
+
+  /// @inheritdoc IGhoAaveSteward
+  address public immutable RISK_COUNCIL;
```

Comment:
this is done within `RiskCouncilControlled` right

Reply:
Updated to work in a cleaner way. Basically was giving me trouble earlier because it's defined in 2 inherited contracts: both IGhoAaveSteward and RiskCouncilControlled. Please let me know if it looks good now!